TECH-OVERSIGHT-COMMITTEE.md

```diff
@@ -96,10 +96,9 @@ The current members of the TOC are shown below.
 | -------------------------------------------------------------- | ---------------- | ---------- | ---------------------------------------------------- | ---------- | --------
 | <img width="30px" src="https://github.com/dprotaso.png">       | Dave Protasowski | VMware     | [@dprotaso](https://github.com/dprotaso)             | 2021-05-26 | 2024     |
 | <img width="30px" src="https://github.com/dsimansk.png">       | David Simansky   | Red Hat    | [@dsimansk](https://github.com/dsimansk)             | 2022-06-01 | 2024     |
-| <img width="30px" src="https://github.com/evankanderson.png">  | Evan Anderson    | VMware     | [@evankanderson](https://github.com/evankanderson)   | Bootstrap  | 2023     |
 | <img width="30px" src="https://github.com/zroubalik.png">      | Zbynek Roubalik  | Red Hat    | [@zroubalik](https://github.com/zroubalik)           | 2022-06-01 | 2024     |
 | <img width="30px" src="https://github.com/psschwei.png"> | Paul Schweigert | IBM | [@psschwei](https://github.com/psschwei) | 2022-09-28 | 2024 |
-
+| <img width="30px" src="https://github.com/psschwei.png"> | Krsna Mahapatra | VMWare | [@kvmware](https://github.com/kvmware) | 2023-06-01 | 2025 |
```

Comment:
as handsome as psschwei.png is, he is not me :)

Reply:
🤣 too quick on the copy/paste!